authors/apps/articles/views.py

```diff
@@ -56,3 +57,25 @@ class DeleteArticle(UpdateAPIView):
     queryset = Article.objects.filter(is_deleted=False)
     serializer_class = DeleteArticleSerializer
     lookup_field = 'slug'
+
+
+class ReadTime(RetrieveAPIView):
+    """
+    class to  get the minutes it take to read an article
+    """
+    serializer_class = GetArticleSerializer
+    queryset = Article.objects.filter(is_deleted=False)
+
+    def retrieve(self, request, slug, *args, **kwargs):
+        try:
+            article_instance = Article.objects.get(slug=slug)
+        except ObjectDoesNotExist:
+            return Response({'error': 'article was not found'}, status.HTTP_404_NOT_FOUND)
+        serializer = self.serializer_class(article_instance)
+        words = serializer.data['body'].split()
+        read = len(words) / 279
```

Comment:
Add a comment to indicate that `279` is the average number of words a person reads per minute